github/spec/features/configure_repos_spec.rb

```diff
@@ -106,27 +106,40 @@
     end
   end
 
-  def given_theres_a_repo(archived: false, full_name: "alphagov/publishing-api", allow_squash_merge: false)
+  def given_theres_a_repo(archived: false,
+                          full_name: "alphagov/publishing-api",
+                          allow_squash_merge: false,
+                          need_production_access_to_merge: false)
     stub_request(:get, "https://api.github.com/orgs/alphagov/repos?per_page=100").
       to_return(headers: { content_type: 'application/json' }, body: [ { full_name: full_name, archived: archived, topics: ["govuk"] }, { full_name: 'alphagov/ignored-for-test', topics: ["govuk"] } ].to_json)
 
     stub_request(:get, "https://api.github.com/repos/#{full_name}/hooks?per_page=100").
       to_return(body: [].to_json, headers: { content_type: 'application/json' })
 
     @repo_update = stub_request(:patch, "https://api.github.com/repos/#{full_name}")
-      .with(
-        body: {
-          allow_merge_commit: true,
-          allow_squash_merge: allow_squash_merge,
-          allow_rebase_merge: false,
-          delete_branch_on_merge: true,
-          name: full_name.split('/').last,
-        }.to_json
```

Comment:
So this `.to_json` wasn't needed?

Reply:
Indeed not. It actually works better without, since we get a clearer diff (not just two blobs of text), and we could do more intelligent matching with `hash_including`.

Reply:
Nice! 👍 I've seen the `.to_json` on this kind of stubbing in various apps, good to know it is advantageous to remove them from now on.